README.md

```diff
@@ -76,12 +76,20 @@ addEventListener("message", async ({data}) => {
 });
 ```
 
+### Worklets
+
+[Worklets](https://html.spec.whatwg.org/multipage/worklets.html#worklets-worklet) (like [CSS Painting API](https://drafts.css-houdini.org/css-paint-api-1/) or [Audio Worklet](https://webaudio.github.io/web-audio-api/#audioworklet)) use the `addModule()` pattern to load a separate file into a Worklet context:
+
+```js
+CSS.paintWorklet.addModule("./my-paint-worklet.js");
+```
+
+The proposal aims to adjust `addModule` analogously to the Worker constructor to accept a Module Block.
+
 ## `ModuleBlock` object model
 
 A module block expression `module { }` evaluates to an instance of a `ModuleBlock` class. Instances are frozen, and the same instance is returned each time for a particular Parse Node, like tagged template literals. `Module` instances have an internal slot [[ModuleBlockHostData]]. This internal slot is filled in by the host in a new host hook called when parsing each module block. (Alternative: A fresh, mutable `ModuleBlock` is returned each time the `module { }` is evaluated, with the same [[ModuleBlockHostData]] each time.) In `import()`, if the parameter has a [[ModuleBlockHostData]] internal slot, it is passed up as is to the dynamic import host hook, and ToString is only called on other values without this slot.
```

Comment:
This paragraph is very much outdated now; maybe it's best to delete it, like you did with the following paragraph.